packages/contracts/.gas-snapshot

```diff
@@ -0,0 +1,16 @@
+test_basefee() (gas: 1068)
+test_hash() (gas: 1067)
+test_number() (gas: 1068)
+test_timestamp() (gas: 1089)
+testCannot_appendCurrentTimestamp() (gas: 4179)
+testCannot_appendEmptyOutput() (gas: 4253)
+testCannot_appendFutureTimestamp() (gas: 4220)
+testCannot_appendOutputIfNotSequencer() (gas: 3491)
+testCannot_appendUnexpectedTimestamp() (gas: 4435)
+testCannot_computePreHistoricalL2BlockNumber() (gas: 2182)
+test_appendingAnotherOutput() (gas: 27894)
+test_computeL2BlockNumber() (gas: 5105)
+test_getL2Output() (gas: 4621)
+test_latestBlockTimestamp() (gas: 1169)
+test_nextTimestamp() (gas: 1572)
+test_constructor() (gas: 6912)
```

Comment:
This is a forge file that we want checked in to see when the cost of a given test changes.